packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -858,28 +858,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const baseSnapshot: ISnapshotTree | undefined = pendingRuntimeState?.baseSnapshot ?? context.baseSnapshot;
 
         let storage = context.storage;
```

Comment:
This assignment should check `pendingRuntimeState` and either use this directly or wrap around `SerializedSnapshotStorage`

Reply:
let me know what you think.

Reply:
looks good.